lib/plugins/aws/package/compile/events/stream/index.js

```diff
@@ -134,6 +157,49 @@ class AwsCompileStreamEvents {
               if (typeof event.stream.enabled !== 'undefined') {
                 Enabled = event.stream.enabled ? 'True' : 'False';
               }
+              if (typeof event.stream.consumer !== 'undefined') {
```

Comment:
Let's just write it as:`if (event.stream.consumer)` (I know that in other cases it's `typeof event.stream.x !== 'undefined'`, but it's bad error-prone pattern, let's not repeat that)

Reply:
Okay, sure.